_sass/includes/footer.scss

```diff
@@ -1,7 +1,32 @@
-.footer.mdl-mega-footer {
+// MDL footer class customizations
+.mdl-mini-footer {
+    padding: 24px;
     color: black;
     background-color: #ffffff;
-    .mdl-mega-footer__heading{
-        color: black;
+    flex-flow: row nowrap;
+    justify-content: center !important;
+    align-items: center;
+    text-align: center;
+  // Custom class for inner div with two child elements logo and paragraph
+  .footer-slogan {
+    display: flex;
+    flex-flow: column wrap !important;
+    justify-content: center !important;
+    align-items: center !important;
+    img {
+      width: 80px;
     }
-}
+    p {
+      margin: 0;
+      font-size: 0.85rem;
+      line-height: 120%;
+      margin: 20px 16px 20px 0;
+    }
+  }
+  // MDL social links customizations
+  .mdl-mini-footer__link-list {
```

Comment:
This is super-minor, but I mention it nonetheless. 

When you cross over the 760px width threshold, the line of text jumps up or down (depending on which direction you are going) just a bit, like one line-height. It seem to be that the flex boxes surrounding the icons changes size, which pushes the text up or drops it down.

<img width="699" alt="Screen Shot 2021-02-12 at 10 38 21 AM" src="https://user-images.githubusercontent.com/68244054/107809808-8bfe1e00-6d20-11eb-96ee-a94d06dbd267.png">
<img width="695" alt="Screen Shot 2021-02-12 at 10 38 31 AM" src="https://user-images.githubusercontent.com/68244054/107809851-99b3a380-6d20-11eb-8fd3-066c0bf51a6c.png">

You might try constraining the height of the flexbox, or something similar, to keep this from happening.  As I said, super minor, but if I noticed it, someone else will.